client/state/data-layer/wpcom/sites/home/layout/index.js

```diff
@@ -6,13 +6,19 @@ import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import { registerHandlers } from 'state/data-layer/handler-registry';
 import { HOME_LAYOUT_REQUEST } from 'state/action-types';
 import { setHomeLayout } from 'state/home/actions';
+import config from 'config';
+
+const cardFeatureFlags = [
+	config.isEnabled( 'editor/before-deprecation' ) && 'editor/before-deprecation',
+];
 
 const requestLayout = ( action ) => {
 	return http(
 		{
 			method: 'GET',
 			path: `/sites/${ action.siteId }/home/layout`,
 			apiNamespace: 'wpcom/v2',
+			query: { flags: encodeURIComponent( cardFeatureFlags.join() ) },
```

Comment:
I don't think we need this level of granularity (one flag per task) and we can have just one flag to group all the experimental/in-progress cards. We can bring back the experimental flag we added in #41145 and D41677-code (and removed in #41855 and D42999-code).

```suggestion
			...( config.isEnabled( 'home/experimental-layout' ) && { query: { experimental: true } } ),
```

Fine to use other word than "experimental" to better denote this will enable the cards currently in development.

Reply:
D43177-code can add back a `dev` flag.

Reply:
See my comment in D43177-code  - if we plumb dev flag through into the other method as suggested it will be an easy fix to change this PR to use that flag instead.

Reply:
Thanks @glendaviesnz. I pushed a changed to D43177-code with your suggestion and updated both D42907-code and this PR. They should be good to go now 👍